mongo/datadog_checks/mongo/dbm/operation_samples.py

```diff
@@ -361,6 +362,7 @@ def _create_operation_sample_payload(
                     "shard": operation_metadata["shard"],
                     "collection": operation_metadata["collection"],
                     "comment": operation_metadata["comment"],
+                    "ns": operation_metadata["ns"],
```

Comment:
should we call this `namesapce`? to be more clear

Reply:
i kinda like to keep it the same as how mongo calls it